callbackScripts/default/displayOff.sh

```diff
@@ -1,2 +1,2 @@
-tvservice -o
-echo "turn off display"
\ No newline at end of file
+vcgencmd display_power 0
```

Comment:
I can not test this, as I don't have a MM setup working right now. But I saw that this solution seemed to work for some of the users. So I'm just trusting you all here :)